conda/recipes/pynvjitlink/meta.yaml

```diff
@@ -1,12 +1,16 @@
 # Copyright (c) 2018-2024, NVIDIA CORPORATION.
 
 {% set data = load_file_data("pyproject.toml") %}
+{% set version = load_file_regex(
+    load_file="pynvjitlink/VERSION",
+    regex_pattern="(?P<value>.*)"
+)[0] %}
```

Comment:
Similar to `scikit-build-core`'s regex, [`conda-build` has a similar feature]( https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#loading-data-from-other-files )

Rewrote the logic above to load from the file `VERSION`

Note this was also need as conda-build can't run `scikit-build-core` first before generating the YAML (and it is probably better that it doesn't). So this reads it directly from the source of truth